appserver/website-bootstrap/explore.js

```diff
@@ -173,73 +177,159 @@ $(document).ready(function(){
     });
 
     document.querySelector("#trends-display-city-time").addEventListener('click', function() {
-        let cityTrendUrl = 'http://18.214.197.203:5000/temporal?woeid=';
+        let cityTrendUrl = 'http://18.214.197.203:5000/temporal_options?';
         let city = document.getElementById('myInput2').value;
         let woeid = citiesMap.get(city);
```

Comment:
should this line go just inside the if(city) {} block?

Reply:
the let woeid = citiesMap.get(city); line?

Reply:
yes

Reply:
yep! fixed